koku/masu/external/downloader/oci_local/oci_local_report_downloader.py

```diff
@@ -145,13 +145,15 @@ def get_manifest_context_for_date(self, date):
             invoice_month = month.strftime("%Y-%m")
             assembly_id = ":".join([str(self._provider_uuid), str(invoice_month)])
             month_file_names = self._extract_names(month)
-            manifest_id = self._process_manifest_db_record(assembly_id, str(month), len(month_file_names), dh._now)
-            monthly_report["manifest_id"] = manifest_id
-            monthly_report["assembly_id"] = assembly_id
-            monthly_report["compression"] = manifest_dict.get("compression", "")
             files_list = [{"key": key, "local_file": self.get_local_file_for_report(key)} for key in month_file_names]
-            monthly_report["files"] = files_list
-            report_manifests_list.append(monthly_report)
+            if len(files_list) > 0:
+                manifest_id = self._process_manifest_db_record(assembly_id, str(month), len(month_file_names), dh._now)
+                monthly_report["manifest_id"] = manifest_id
+                monthly_report["assembly_id"] = assembly_id
+                monthly_report["compression"] = manifest_dict.get("compression", "")
+                monthly_report["files"] = files_list
+                report_manifests_list.append(monthly_report)
+        LOG.info(f"Report Manifests List: {str(report_manifests_list)}")
```

Comment:
Inside f-strings, `str()` is called by default on values inside expressions.
```suggestion
        LOG.info(f"Report Manifests List: {report_manifests_list}")
```